hastory-server/src/Hastory/Server/OptParse.hs

```diff
@@ -0,0 +1,155 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Hastory.Server.OptParse where
+
+import Control.Monad
+import Data.Maybe
+import qualified Data.Text as T
+import Data.Yaml
+import qualified Env
+import GHC.Generics (Generic)
+import Options.Applicative as OptParse
+import qualified Options.Applicative.Help as OptParse (string)
+import Path
+import Path.IO
+import Text.Read
+import YamlParse.Applicative as YamlParse
+
+data Instructions = Instructions Dispatch Settings deriving (Show, Eq, Generic)
+
+newtype Dispatch = DispatchServe ServeSettings deriving (Show, Eq, Generic)
+
+data Settings = Settings deriving (Show, Eq, Generic)
+
+data ServeSettings = ServeSettings {serveSettingsPort :: Int, serveSettingsLogFile :: Path Abs File, serveSettingsKeyFile :: Path Abs File} deriving (Show, Eq, Generic)
```

Comment:
These don't really need to be all on one line...
For a quick fix, first run `hindent`, then `ormolu`.

Reply:
Hm.. - I wonder if I am doing something wrong. I go into `nix-shell` and I run `pre-commit run --all-files` before every commit, and I can see that it runs `ormolu` and `hindent`, which all pass. I did a bit of manual reformatting to address this specific issue, but I think the intent is to avoid that.